Author: Yury Kudryashov
Reason: other KDE modules try to install polkit action files into
$kdelibs/share/polkit-1/actions
--- a/cmake/modules/FindPolkitQt-1.cmake.old	2010-08-12 16:13:45.000000000 +0400
+++ b/cmake/modules/FindPolkitQt-1.cmake	2010-08-12 16:13:56.000000000 +0400
@@ -101,4 +101,4 @@
     endif (NOT PC_POLKITQT-1_PREFIX STREQUAL CMAKE_INSTALL_PREFIX)
 endif (POLKITQT-1_FOUND)
 
-set(POLKITQT-1_POLICY_FILES_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/share/polkit-1/actions)
+set(POLKITQT-1_POLICY_FILES_INSTALL_DIR share/polkit-1/actions)
